src/test/java/io/xstefank/wildfly/bot/PRCommitsQuantityCheckTest.java

```diff
@@ -22,9 +24,11 @@
 public class PRCommitsQuantityCheckTest {
 
     private String wildflyConfigFile;
+    private JsonHandler.JsonHandlerBuilder jsonHandlerBuilder;
 
     @BeforeEach
-    void setUp() {
+    void setUp() throws IOException {
+        jsonHandlerBuilder = new JsonHandler.JsonHandlerBuilder("pr-template.json");
```

Comment:
I would suggest making this a static method 
```suggestion
        jsonHandlerBuilder = new JsonHandler.builder("pr-template.json");
```

you also don't need to keep it in a field